keyboards/kv/revT/readme.md

```diff
@@ -0,0 +1,19 @@
+KVT
+===
+
+![KVT default layout](https://imgur.com/nPwxzdU)
+
+A open source southpaw keyboard. The Default layer is a standard 104 ANSI keyboard just the numpad is on the left.
+There is a fn layer (its not in the layout image but in the keymap file) it adds media keys, f13-f24, and mouse keys.
+
+[More info on KVT github](https://github.com/Hybrid65/KVT)
+
+Keyboard Maintainer: [Neil Feagan](https://github.com/Hybrid65)
+Hardware Supported: KVT, Proton C  
+Hardware Availability: [List of vendors](https://qmk.fm/proton-c)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make kv/revT:default
```

Comment:
"revT" should only contain lowercase chars -- I'd suggest changing all such references and filenames to either "revt" or "rev_t".